README.md

```diff
@@ -27,173 +26,116 @@ CockroachDB is currently in alpha. See our
-$ docker-machine create --driver virtualbox default
-# if the VM exists but isn't running, start it:
-$ docker-machine start default
-# set up the environment for the docker client:
-$ eval $(docker-machine env default)
-```
-Other operating systems will have a similar set of commands. Please check Docker's documentation for more info.
-
-Pull the CockroachDB Docker image and drop into a shell within it:
-```bash
-docker pull cockroachdb/cockroach
-docker run -p 26257:26257 -p 8080:8080 -t -i cockroachdb/cockroach shell
-# root@82cb657cdc42:/cockroach#
-```
+## Quickstart 
+
+1.  [Install Cockroach DB](https://www.cockroachlabs.com/docs/install-cockroachdb.html).
+
+2.  [Start a local cluster](https://www.cockroachlabs.com/docs/start-a-local-cluster.html) with three nodes running on different ports:
+
+    ```shell
+    $ ./cockroach start --insecure &
+    $ ./cockroach start --insecure --store=node2-data --port=26258 --http-port=8081 --join=localhost:26257 &
+    $ ./cockroach start --insecure --store=node3-data --port=26259 --http-port=8082 --join=localhost:26257 &
+    ```
+
+3.  [Start the built-in SQL client](https://www.cockroachlabs.com/docs/use-the-built-in-sql-client.html) as an interactive shell:
+
+    ```shell
+    $ ./cockroach sql --insecure
+    # Welcome to the cockroach SQL interface.
+    # All statements must be terminated by a semicolon.
+    # To exit: CTRL + D.
+    ```
+
+4. Run some [CockroachDB SQL statements](https://www.cockroachlabs.com/docs/learn-cockroachdb-sql.html):
+
+    ```shell
+    root@:26257> CREATE DATABASE bank;
+    CREATE DATABASE
 
-### Bootstrap and talk to a single node
-
-Note: If you’re using Docker as described above, run all the commands described below in the container’s shell.
+    root@:26257> SET DATABASE = bank;
+    SET
 
-Setting up Cockroach is easy, but starting a test node is even easier. All it takes is running:
+    root@:26257> CREATE TABLE accounts (id INT PRIMARY KEY, balance DECIMAL);
+    CREATE TABLE
 
-```bash
-./cockroach start --insecure &
-```
+    root@26257> INSERT INTO accounts VALUES (1234, DECIMAL '10000.50');
+    INSERT 1
 
-Verify that you're up and running by visiting the cluster UI. If you're running
-without Docker (or on Linux), you'll find it at
-[localhost:8080](http://localhost:8080); for OSX under Docker, things are a
-little more complicated and you need to run `docker-machine ip default` to get
-the correct address (but the port is the same).
+    root@26257> SELECT * FROM accounts;
+    +------+----------+
+    |  id  | balance  |
+    +------+----------+
+    | 1234 | 10000.50 |
+    +------+----------+
+    ```
 
-##### Built-in client
+4. Checkout the admin UI by pointing your browser to `http://<localhost>:26257`.
 
-Now let's talk to this node. The easiest way to do that is to use the `cockroach` binary - it comes with a built-in sql client:
+5. Learn how to [secure a cluster](https://www.cockroachlabs.com/docs/secure-a-cluster.html).
```

Comment:
How about make this less daunting: "CockroachDB makes it easy to [secure a cluster]"